packages/syft/src/syft/service/enclave/enclave_service.py

```diff
@@ -148,74 +48,6 @@ def setup_enclave_for_code_execution(
         result = context.node.get_service("usercodeservice").submit(root_context, code)
```

Comment:
I know we just merged the other PR but should we consider changing the `root_context` to using `node.root_client` for uniformity?

Reply:
We ideally don't want to use the client inside the service methods. This was a temporary workaround for v1. In the next iteration we'll find a way to create jobs without using clients.